src/commands/createStaticWebApp/StaticWebAppCreateStep.ts

```diff
@@ -31,15 +53,33 @@ export class StaticWebAppCreateStep extends AzureWizardExecuteStep<IStaticWebApp
             sku: context.sku,
             location: (await LocationListStep.getLocation(context)).name
         };
-
         const creatingSwa: string = localize('creatingSwa', 'Creating new static web app "{0}"...', newName);
         progress.report({ message: creatingSwa });
         ext.outputChannel.appendLog(creatingSwa);
         context.staticWebApp = await context.client.staticSites.beginCreateOrUpdateStaticSiteAndWait(nonNullValueAndProp(context.resourceGroup, 'name'), newName, siteEnvelope);
         context.activityResult = context.staticWebApp as AppResource;
+
+        if (context.logicApp) {
+            //link backends only if SWA called with LA
```

Comment:
instead of the comment can we extract the content.logicApp into a varthat is named properly so that thise comment is not needed anymore